python/lsst/daf/butler/registry/tests/_registry.py

```diff
@@ -95,6 +101,8 @@ def makeRegistryConfig(self) -> RegistryConfig:
         config = RegistryConfig()
         if self.collectionsManager:
             config["managers"]["collections"] = self.collectionsManager
+        if self.datasetsManager:
+            config["managers"]["datasets"] = self.datasetsManager
```

Comment:
If this is a `Config` you can actually write:

```python
config["managers", "datasets"] = self.datasetsManager
```
because the tuple is unpacked into the hierarchy (but what you've written also works of course).

Reply:
Indeed, I just copied two lines above without much brain activity involved 🙂